packaging/technical-addon/packaging-scripts/update-ta-version.sh

```diff
@@ -7,7 +7,7 @@ fi
 # for sed command, the capturing group for version is what will be printed (returned), but it'll print every line, so only grab first
 OLD_TA_VERSION="$(sed -n 's/^version = \(.*$\)/\1/p' "$APP_CONFIG" | head -n 1)"
 if [ -z "$TA_VERSION" ]; then
-    TA_VERSION=$(echo "$OLD_TA_VERSION" | awk -F. '{printf "%d.%d.%d", $1,$2,$3+1}')
+    TA_VERSION=$(echo "$OLD_TA_VERSION" | awk -F. '{printf "%d.%d.%d", $1,$2+1,0}')
```

Comment:
the current version is 1.4.4 , eventually we can change 3rd digit to $3 but for now, we can hardcode the patch to 0 in case we released a patch version.
For now, patches can still be overwritten by setting  `TA_VERSION` env variable when calling the make targets.